gradle.properties

```diff
@@ -1,7 +1,9 @@
 projectVersion=4.0.0-SNAPSHOT
 
-# This prevents the Grails Gradle Plugin from unnecessarily excluding slf4j-simple in the generated POMs
-# https://github.com/grails/grails-gradle-plugin/issues/222
+grailsVersion=7.0.0-SNAPSHOT
+quartzVersion=2.5.0
+slf4jVersion =2.0.16
+
 slf4jPreventExclusion=true
 
 websiteUrl=https://grails.github.io/grails-quartz/latest/
```

Comment:
weburl, issuetrackerurl, and vcsurl should be removable now that you're using grailsPublish

Reply:
deleted